Tests/UtilityTests/ArgumentParserTests.swift

```diff
@@ -367,6 +368,21 @@ class ArgumentParserTests: XCTestCase {
         }
     }
 
+    func testPathArgument() {
+        // relative path
+        do {
+            let actual = try! SwiftPMProduct.TestSupportExecutable.execute(["absolutePath", "some/path"]).chomp()
+            let expected = currentWorkingDirectory.appending(RelativePath("some/path")).asString
+            XCTAssertEqual(actual, expected)
+        }
+
+        // absolute path
+        do {
+            let actual = try! SwiftPMProduct.TestSupportExecutable.execute(["absolutePath", "/bin/echo"]).chomp()
+            XCTAssertEqual(actual, "/bin/echo")
```

Comment:
Actually this can accept both paths so there should be no failures. But we should still add some edge cases for eg if `../foo` and `./foo` resolves correctly etc.